.vib/flink/cypress/cypress/integration/flink_spec.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="cypress" />
+
+it('should show available task slots', () => {
+  cy.visit('/')
+  cy.contains('div.field', 'Total Task Slots').within(() => {
+    cy.contains('span', /^\d+$/).should(($span) => {
+      const num = parseInt($span.text());
+      expect(num).to.be.greaterThan(0); // Asserts that the number in the second span is greater than 0
+    });
+  });
+})
+
+it('should show completed jobs', () => {
```

Comment:
I would merge these two into a single one, the use case is basically the same (checking info in the main page)

Reply:
In this case IMO it's better to have both tests separated, since they are separated cases in terms of application logic. Although they check the same page, on ensures that jobs are working meanwhile the other one checks the connection of the Flink Jobmanager and Taskmanager.